tests/test_core.py

```diff
@@ -15,56 +21,101 @@
     IOConfigInterface,
 )
 
+
 # pragma pylint: disable=R0903
 #  Disable "too few public methods" for test cases - most test files will be classes used for
 #  grouping and then individual tests alongside these
 
 
 class TestComponent:
-    # Test a set of what interface passing succeeds and fails
+    """
+    Test a set of what interface passing succeeds and fails.
+    """
+
     @staticmethod
     def test_componentkind_interface_map_behaviour() -> None:
+        """
+        The interface method returns the interface supported by that Component.
```

Comment:
These are missing line-breaks after the first sentence.
_However_, pydocstyle ignores tests by default, and I have already address this is 2dd8444 which can be a future merge